zod/user.ts

```diff
@@ -0,0 +1,30 @@
+import { z } from "zod";
+
+export const userLoginSchema = z.object({
```

Comment:
rename this from user.ts to something else to help not create a confusion, it has schema for newJob as well as user which might be confusing with respect to its name

Reply:
thankss

Reply:
seperated the user schema and job schema in different files